kubernetes/assets/monitors/monitor_pod_crashloopbackoff.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name": "[kubernetes] Pod {{pod_name.name}} is CrashloopBackOff on namespace {{kube_namespace.name}}",
 	"type": "query alert",
-	"query": "avg(last_30m):default(avg:kubernetes.containers.state.waiting{reason:crashloopbackoff} by {kube_namespace,pod_name}, 0) >= 1",
+	"query": "max(last_10m):max:kubernetes.containers.state.waiting{reason:crashloopbackoff} by {kube_namespace,pod_name} >= 1",
```

Comment:
Looks like this metric is available with KSM as well. Should we use that in order to be consistent in using KSM everywhere possible? Not sure if there's a reason you chose not to go with KSM here.